homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -299,3 +321,75 @@ def _async_update_shade_from_group(self):
             return
         self._async_process_new_shade_data(self.coordinator.data[self._shade.id])
         self.async_write_ha_state()
+
+
+class SilhouettePowerViewShade(PowerViewShade):
+    """Class to represent a Silhouette (Type 23) blind."""
+
+    def __init__(self, coordinator, device_info, room_name, shade, name):
+        """Initialize the shade."""
+        super().__init__(coordinator, device_info, room_name, shade, name)
+        self._max_tilt = 32767
+        self._attr_current_cover_tilt_position = 0
+        self._attr_supported_features |= (
+            CoverEntityFeature.OPEN_TILT
+            | CoverEntityFeature.CLOSE_TILT
+            | CoverEntityFeature.STOP_TILT
+            | CoverEntityFeature.SET_TILT_POSITION
+        )
+
+    async def async_open_cover_tilt(self, **kwargs):
+        """Open the cover tilt."""
+        self._async_update_from_command(await self._shade.tilt_open())
+
+    async def async_close_cover_tilt(self, **kwargs):
+        """Close the cover tilt."""
+        self._async_update_from_command(await self._shade.tilt_close())
+
+    async def async_set_cover_tilt_position(self, **kwargs):
+        """Move the cover tilt to a specific position."""
+        if ATTR_TILT_POSITION not in kwargs:
+            return
+        target_hass_tilt_position = kwargs[ATTR_TILT_POSITION]
+        current_hass_position = hd_position_to_hass(
+            self._current_hd_cover_position, MAX_POSITION
+        )
+        steps_to_move = (
+            current_hass_position + 10
+        )  # tilting is fast, so just add a bit on
```

Comment:
I'm not sure this is right as I think `_current_hd_cover_position` is not the tilt position here

Reply:
i renamed `_current_cover_position` to `_current_hd_cover_position` because it was confusing. arguably outside of the scope of the PR but was being a good boy-scout and tidying up.
Will take your lead on next steps

Reply:
I think you want to use the tilt position (aka POSKIND_VANE) here to calculate steps_to_move and not the `_current_hd_cover_position`

Reply:
The vane tilt is do fast I just added 10 arbitrarily as it didn't seem worth calculating the time difference between degrees

Reply:
Makes sense.  I don't have any that tilt so I wasn't aware they had to function in unison